mpy-cross/mpy-cross.vcxproj

```diff
@@ -100,4 +103,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
This is a non-related change.... @stinos what is your stance on new-line-at-end-of-file for these files?

Reply:
In this case: no new line; reason being that if these files get edited through Visual Studio the newline gets actively removed by it.